talk_video_maker/videos.py

```diff
@@ -104,8 +104,8 @@ def exported_audio(self, format, sample_rate=None):
         streams = filter_streams(streams, {'audio'}, 'aformat', args)
         return AVObject(streams, acodec='pcm_s16le', format='wav')
 
-    def muted(self):
-        streams = [s for s in self.streams if s.type != 'audio']
+    def muted(self, mute_type='audio'):
+        streams = [s for s in self.streams if s.type != mute_type]
         return AVObject(streams)
```

Comment:
This looks useful, but the word `mute` refers only to sound. Could you rename the method to `remove_streams`, for example? (And make `mute` call `remove_streams(type='audio')`)

Reply:
OK, I refactored it as without_streams, to be consistent with other command (cropped, muted). I've added muted as a convenience function to remove audio stream.